zenoh/src/net/runtime/orchestrator.rs

```diff
@@ -1253,4 +1260,18 @@ impl Runtime {
         let hat_code = tables.hat_code.clone();
         hat_code.links_info(&tables)
     }
+
+    pub(crate) fn get_router_path(&self, src: ZenohId, dst: ZenohId) -> Option<NetworkPath> {
```

Comment:
I don't see why this is in orchestrator.rs`. I'd rather have admin space directly call hat functions like it's done for `routers_linkstate_data` for coherency.